features/step_definitions/networking.rb

```diff
@@ -437,12 +437,18 @@
 Given /^the cluster network plugin type and version and stored in the clipboard$/ do
   ensure_admin_tagged
   _host = node.host
-
-  step %Q/I run the ovs commands on the host:/, table([[
-    "ovs-ofctl dump-flows br0 -O openflow13 | grep table=253 | sed -n -e 's/^.*note://p'"
-  ]])
-  if @result[:success]
-    of_note = @result[:response].chomp
+  
+  if env.version_lt("3.11", user: user)
```

Comment:
master branch is only for 4.x so conditions for 3.x are not needed. If yo want to fix 3.x, then `v3` branch needs to be updated.

Reply:
@akostadinov Yea, Thanks for pointing out that. Right. I will re-look at it